src/gstvimbasrc.c

```diff
@@ -377,21 +377,21 @@ static void gst_vimbasrc_class_init(GstVimbaSrcClass *klass)
         g_param_spec_int(
             "offsetx",
             "OffsetX feature setting",
-            "Horizontal offset from the origin to the region of interest (in pixels).",
+            "Horizontal offset from the origin to the region of interest (in pixels). If no explicit value is passed the RoI will be centered in the sensor.",
```

Comment:
Update string to explain the magic value `-1`.

```suggestion
            "Horizontal offset from the origin to the region of interest (in pixels). If -1 is passed the ROI will be centered in the sensor along the horizontal axis.",
```